README.md

```diff
@@ -1,2 +1,142 @@
-# consume-more-stuff
-A custom CMS for personal product sale
+# Consume More Stuff
+> A custom CMS for personal product sale
+
+## Introduction
+Build a Content Management System using:
+
+- __Backend__
+    - **Express** as the application server
+    - **Postgres** as the database server
+    - **Sequelize.js** as the ORM
+    - **passport.js** for Authentication and Authorization
+
+- __Frontend__
+	- **React.js** or **AngularJS** as FE framework
+	- **Sass** for styling
+	- **Webpack** for bundling	
```

Comment:
Just to reconfirm that they would use a generator for Angular or React (create-react-app). We don't want to nuke minds with teaching webpack. 

If anything I would say they would be learning or "doing" webpack. They're using a generator/boilerplate that implements webpack. Saying they know webpack would is false.

Reply:
Yes webpack will be using what is provided by the generators.